packages/fxa-payments-server/src/components/PaymentForm/index.tsx

```diff
@@ -317,4 +249,58 @@ const WrappedPaymentForm = (props: PaymentFormProps) => (
   </Elements>
 );
 
+export const SMALL_DEVICE_RULE = '(max-width: 520px)';
+export const SMALL_DEVICE_LINE_HEIGHT = '40px';
+export const DEFAULT_LINE_HEIGHT = '48px';
+
+export function mkStripeElementStyles(useSmallDeviceStyles: boolean) {
```

Comment:
Second refactor: I noticed that we were defining & mutating these styles on successive renders. Figured this would be easier to just define as a function that outputs the styles.